src/malli/generator.cljc

```diff
@@ -40,16 +46,23 @@
         gen (-> schema m/children first (generator options))]
     (gen/fmap f (gen/vector-distinct gen {:min-elements min, :max-elements max, :max-tries 100}))))
 
+(defn -or-gen [schema options]
+  (gen/one-of (keep #(some->> (-recursion-options % options) (generator %)) (m/children schema options))))
+
+(defn -multi-gen [schema options]
+  (gen/one-of (keep #(some->> (-recursion-options (last %) options) (generator (last %))) (m/map-entries schema options))))
+
 (defn -map-gen [schema options]
   (let [entries (m/map-entries schema)
-        value-gen (fn [k s] (gen/fmap (fn [v] [k v]) (generator s options)))
+        options' (-recursion-options schema options)
```

Comment:
Perhaps this is better filed as a bug with them, but just worth noting that the "prime" convention isn't well supported in some editors' understanding of the clojure grammar. Emacs will attempt to convert `options'` into `options '` causing the expression to rebind `options` to the quoted expression `'(-recursion-options schema options)`. Your haskell is showing ;)